bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoVacuumHandler.java

```diff
@@ -51,19 +52,63 @@
  *
  * @author Marcel Verpaalen - Initial contribution
  */
+@NonNullByDefault
 public class MiIoVacuumHandler extends MiIoAbstractHandler {
     private final Logger logger = LoggerFactory.getLogger(MiIoVacuumHandler.class);
-
     private ExpiringCache<String> status;
     private ExpiringCache<String> consumables;
     private ExpiringCache<String> dnd;
     private ExpiringCache<String> history;
-    private String lastHistoryId;
+    private String lastHistoryId = "";
+    private int inCleaning;
```

Comment:
Why was this made a field?

Reply:
Indeed... Not really for this PR... For the next one...
I extracted these changes from the upcoming big pr to separate these many cleanup changes from the real functionality changes.

https://community.openhab.org/t/xiaomi-robot-vacuum-binding/31317/1321